samples/tfm_integration/tfm_secure_partition/README.rst

```diff
@@ -0,0 +1,49 @@
+.. _tfm_secure_partition:
+
+TF-M Secure Partition Sample
+############################
+
+Overview
+********
+
+A Secure Partition is an isolated module that resides in TF-M. It exposes a number of functions or "secure services" to other partitions and/or to the non-secure firmware.
+TF-M already contains standard partitions such as crypto, protected_storage, or firmware_update, but it's also possible to create your own partitions.
+
+This sample creates a dummy secure partition and secure service for TF-M and instructs the TF-M build system to build it into the secure firmware.
+The dummy secure service is then called in the main file (in the non-secure firmware).
+
+Everything in the "partition" directory is built by the TF-M build system, refer to :ref:`tfm_build_system` for more details.
```

Comment:
I thought this was referring to a directory literally named "partition". Can we make this clearer?

Reply:
I've restructured the text, hopefully this is clearer.